src/pages/HomePage.jsx

```diff
@@ -1,15 +1,58 @@
-import { Input, Signup, Layout } from '../components';
+import { Layout, CustomButton } from "../components";
+import { Link } from "react-router-dom";
+
+import imgLogo from "../assets/logo.png";
+import { Typography, Grid } from "@mui/material";
+
 
```

Comment:
stick to import conventions 
firstly import from node modeules
empyt line
import local files

Reply:
fixed